WebDriverAgentLib/Commands/FBAlertViewCommands.m

```diff
@@ -44,7 +45,13 @@ + (NSArray *)routes
 + (id<FBResponsePayload>)handleAlertAcceptCommand:(FBRouteRequest *)request
 {
   FBSession *session = request.session;
-  if (![[FBAlert alertWithApplication:session.application] acceptWithError:nil]) {
+  NSString *name = request.parameters[@"name"];
+
+  if ([name length] != 0) {
+    if (![[FBAlert alertWithApplication:session.application] clickAlertButton:name error:nil]) {
+      return FBResponseWithStatus(FBCommandStatusNoAlertPresent, nil);
```

Comment:
I get your point, but I'm not sure how to fix it - would you want to respond with FBCommandStatusNoAlertPresent and a more detailed error message, or do you want to return a different status code - something along the lines of Element not found?